README.md

```diff
@@ -1 +1,149 @@
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Enumerable] <a name="about-project"></a>
+
+>This repo is a school library tool that allows you to record what books are in the library and who borrows them, the project focused on practicing the main four concepts in oop using ruby.
+
+**[your_project__name]**  
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a> 
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://ruby.org/">Ruby</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features" /> 
+**[useing module]**
+**[useing all? any? filter?]**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo" />
+
+- [Live Demo Link]()
+
+## 💻 Getting Started <a name="getting-started" />
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Ruby
+- Git
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+  cd "to where you want to install your project"
+
+ https://github.com/AhmadiGu/OOP_Libaray
+```
+
+### Install
+
+Install this project with:
+
+```
+ cd my-project
+  gem install
+```
+
+ 
+
+## 👥 Authors <a name="authors" /> 
+
+👤 **Gulalai Ahmadi**
+
+- GitHub: [GitHub Profile](https://github.com/AhmadiGu)
+- LinkedIn: [LinkedIn Profile](https://ahmadigu.github.io/Desktop_Portfolio/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[School libaray]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing" />
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support" />
+
+If you like this project support it through staring the project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements" />
+
+I would like to thank Microverse for idea of the project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```

Comment:
- Hey, let's kindly remove this placeholder text as the text is there to help you update the readme template to suit your project needs. let's remove to make the readme more professional.